attrs_strict/_error.py

```diff
@@ -89,3 +90,17 @@ def __str__(self):
         )
 
         return self._render(error)
+
+
+def _is_newtype(type_):
+    """Checks if `type` is :func:`typing.NewType`."""
+    return (hasattr(type_, "__qualname__") and
+            type_.__qualname__ == "NewType.<locals>.new_type")
+
+
+def _format_type(type_):
+    """Returns a string to pretty display a given `type_` to the user."""
+    if _is_newtype(type_):
```

Comment:
can you inline these two ?

Reply:
That would exceed 80 characters per line, which seems to be the maximum line length in this project.

Reply:
That would exceed 80 characters per line, which seems to be the maximum line length in this project.

Reply:
That's fine, you can use `black` to format it.

Reply:
Yes, black make things easy. However, your black is configured with maximum 80 characters per line in the pyproject.toml .

Reply:
Yes, that's the recommendation by pep8.